ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-master-presubmits.yaml

```diff
@@ -393,8 +425,6 @@ presubmits:
             configMapKeyRef:
               key: master.yaml
               name: ci-operator-openshift-openshift-ansible
-        - name: HOME
```

Comment:
Why do you need this for unit tests @vrutkovs ?

Reply:
One of the test is verifying syntax of ansible playbook, which is ansible call. Ansible wants `$HOME` to store facts

Reply:
Let's not throw anything away -- why don't you set a temp `$HOME` in your test? Origin tests do that all the time.